code/game/objects/structures/window.dm

```diff
@@ -7,7 +7,7 @@
 
 	layer = SIDE_WINDOW_LAYER
 	anchored = 1.0
-	atom_flags = ATOM_FLAG_NO_TEMP_CHANGE | ATOM_FLAG_CHECKS_BORDER
```

Comment:
This flag only exists for windows as far as I can tell - what is removing it meant to fix?

Reply:
Seems to be about forbidding movement onto low walls. Probably the correct fix is to set this if `dir` is a cardinal, and unset it for full-tile windows (don't recall if they are `dir = 0` or `dir = 5`).

Reply:
Tested this locally and it doesn't seem to impact window behavior at all.